src/components/forms/message.js

```diff
@@ -4,26 +4,31 @@ const messages = {
   confirm: {
     type: "success",
     text:
-      "A confirmation message was sent to your email, click the link there to continue."
+      "Un message de confirmation a été envoyé sur votre boite mail, validez votre email pour continuer."
```

Comment:
Sorry I missed that in my first review. The sentence is weird. Better : "Un message contenant un lien d'activation a été envoyé sur votre boite mail. Cliquez sur ce lien pour terminer l'inscription."

Reply:
thanks for review, it looks better, true